packages/ia-components/sandbox/tiles/TileComponent.jsx

```diff
@@ -58,6 +58,7 @@ export default class TileComponent extends IAReactComponent {
             const collectionSize = member.item_count; //TODO really hard to get https://github.com/internetarchive/dweb-archive/issues/91
             const classes = ['item-ia']
             if (isCollection) classes.push('collection-ia');
+            if (member.crawl) classes.push('crawl-'+member.crawl.level); // Whether crawled or not and at what level
```

Comment:
**Suggestion**: replace string concatenation with a template literal.

Reply:
Rejected:  Same response as to the previous one, why are you suggesting changes that make zero functional difference, that just happen to be your preference and are actually slower https://stackoverflow.com/questions/29055518/are-es6-template-literals-faster-than-string-concatenation